### ISSUE TITLE
script to move a measure from one table to another

### ISSUE THREAD

**Issue by o-o00o-o** [otykier/TabularEditor#234]
Hi Daniel, what is the best scripting method to use if I have a measure that I want to change its table? I've looked through the TOMWrapper but can't see an obvious answer. 
Thanks
Brett

**Comment by otykier** [comment]
It's possible, but it's not pretty:

```csharp
UI.UIController.Current.Handler.Actions.MoveObjects(
    Model.Tables["Reseller Sales"].Measures,
    Model.Tables["Internet Sales"]
);
```

The `MoveObjects` method takes two arguments: The first is a collection of IFolderObjects (that is, measures, columns or hierarchies), the second is the destination table.

This is going to improve in a future version, once #201 has been implemented (as UI actions such as drag/drop should also be recordable and generate more "pretty" code).

**Comment by o-o00o-o** [comment]
ok thanks. and if I wanted to clone a column to a different table I guess I would use Clone to a dummy name and then use the above method to move (doesn't seem to be a CloneTo method for Columns for some reason?).

These operations will all update all references I presume?

**Comment by o-o00o-o** [comment]
Is the TOMWrapper doc out of date? it says they are collection of IDetailObject but this cannot be found.

**Comment by otykier** [comment]
Yes, the doc is out of date. IDetailObject was renamed to IFolderObject at one point. I'm not sure that the formula fix-up supports objects moving between tables (for measures, we shouldn't need to update anything, as measure references should not be qualified with the table name). Please test it and raise an issue if it doesn't work as expected.
